From a6f15b4974293e685af773d7f536f5406c3548db Mon Sep 17 00:00:00 2001
From: Alexander Heinisch <alexander.heinisch@siemens.com>
Date: Tue, 5 Aug 2025 17:53:05 +0200
Subject: [PATCH] Fixing iteration on autoexec.ipxe.

Signed-off-by: Alexander Heinisch <alexander.heinisch@siemens.com>
---
 src/interface/efi/efi_autoexec.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/interface/efi/efi_autoexec.c b/src/interface/efi/efi_autoexec.c
index 73ba5df33..a5feca83a 100644
--- a/src/interface/efi/efi_autoexec.c
+++ b/src/interface/efi/efi_autoexec.c
@@ -201,7 +201,7 @@ int efi_autoexec_load ( void ) {
 			DBGC ( device, "EFI %s discarding zero-length %s\n",
 			       efi_handle_name ( device ), image->name );
 			unregister_image ( image );
-			return -ENOENT;
+			continue;
 		}
 
 		DBGC ( device, "EFI %s loaded %s (%zd bytes)\n",
-- 
2.39.5

